sdk/go/common/env/env.go

```diff
@@ -129,6 +129,9 @@ var DisableSecretCache = env.Bool("DISABLE_SECRET_CACHE",
 var ParallelDiff = env.Bool("PARALLEL_DIFF",
 	"Enable running diff calculations in parallel.")
 
+var Refresh = env.String("REFRESH",
```

Comment:
We have a bool type here, and we should use it.
```suggestion
var Refresh = env.Bool("REFRESH",
```

Reply:
Ian is right. Bool here should be fine, the env system will translate `--refresh=true` to the true value.

Reply:
hmm, ok I will try it. I thought I remember it failing, but maybe it was a test I added and not an original test

Reply:
```
Error: cmd/pulumi/operations/destroy.go:413:29: cannot use env.Refresh.Value() (value of type bool) as string value in argument to cmd.PersistentFlags().StringVarP
Error: cmd/pulumi/operations/preview.go:646:29: cannot use env.Refresh.Value() (value of type bool) as string value in argument to cmd.PersistentFlags().StringVarP
Error: cmd/pulumi/operations/up.go:715:29: cannot use env.Refresh.Value() (value of type bool) as string value in argument to cmd.PersistentFlags().StringVarP```

Reply:
Oh gods I forgot about "always", nice spot